kuma/static/js/components/mathml.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * vanilla version of the function at:
+ * https://github.com/mdn/kuma/blob/master/kuma/static/js/wiki.js#L252
```

Comment:
When linking to lines on GitHub I recommend linking to a specific hash instead of linking to the tip of a branch. If the moves lines in the file, the link still points to the intended statement. Downside is ofc that it might be outdated. But at least you know what to look for.
To turn a branch-link into a commit sha-link press `y` while viewing that file within GitHub

Reply:
Agreed. To be honest, though, I should have just added this as a comment to the PR. This will not be very useful for very long.